Sources/Testing/Running/Configuration.swift

```diff
@@ -178,14 +178,35 @@ public struct Configuration: Sendable {
 
   // MARK: - Event handling
 
-  /// Whether or not events of the kind
-  /// ``Event/Kind-swift.enum/expectationChecked(_:)`` should be delivered to
-  /// this configuration's ``eventHandler`` closure.
+  /// A type describing options to use when delivering events to this
+  /// configuration's event handler
+  public struct EventHandlingOptions: Sendable {
+    /// Whether or not events of the kind ``Event/Kind-swift.enum/issueRecorded(_:)``
+    /// containing issues with warning (or lower) severity should be delivered
+    /// to the event handler of the configuration these options are applied to.
+    ///
+    /// By default, events matching this criteria are delivered to event
+    /// handlers unless the entry point specifies an event stream version which
+    /// predates the introduction of warning issues, in which case this is
+    /// disabled to maintain legacy behavior.
+    public var isWarningIssueEventDeliveryEnabled: Bool = true
```

Comment:
```suggestion
    public var isWarningIssueRecordedEventEnabled: Bool = true
```

Reply:
I like that (and the other suggested property name)